sdv/constraints/base.py

```diff
@@ -126,7 +126,8 @@ def _validate_inputs(cls, **kwargs):
                 f'Invalid values {invalid_vals} are present in {article} {constraint} constraint.'
             ))
 
-        raise MultipleConstraintsErrors(errors)
+        if errors:
```

Comment:
Unrelated.

Reply:
Good catch! maybe we should add a test for this case